akka-http2-support/src/main/scala/akka/http/impl/engine/http2/AlpnSwitch.scala

```diff
@@ -18,35 +18,29 @@ import akka.stream._
 /** INTERNAL API */
 @InternalApi
 private[http] object AlpnSwitch {
```

Comment:
Since it's now more generic, we should probably give it a more generic name - `ProtocolSwitch`?

Reply:
Good point. Done.